.github/CODEOWNERS

```
@@ -0,0 +1,7 @@
+* @hugomrdias @rvagg
+
+/docs/src/content/cookbooks @FilOzone/fil-b
+/docs/src/content/core-concepts @FilOzone/fil-b 
+/docs/src/content/introduction @FilOzone/fil-b 
+/docs/src/content/resources @FilOzone/fil-b 
+/docs/src/content/developer-guides @FilOzone/fil-b @hugomrdias @rvagg
```

```suggestion
/docs/src/content/cookbooks @FilOzone/FIL-B
/docs/src/content/core-concepts @FilOzone/FIL-B 
/docs/src/content/introduction @FilOzone/FIL-B 
/docs/src/content/resources @FilOzone/FIL-B 
/docs/src/content/developer-guides @FilOzone/FIL-B @hugomrdias @rvagg
```

github is complaining about this in the review pane